config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -5222,6 +5222,19 @@ rubrics.integration.token-secret=12345678900909091234
 # default: empty
 # turnitin.oc.apiKey=mysupersecret
 
+# Maximum delay between retires after recoverable errors
+# default: 240
+# turnitin.oc.max.retry.minutes=240
+
+# Maximum number of retries for recoverable errors
+# default: 16
+# turnitin.oc.max.retry=16
+
+# Skips any delays intended to reduce traffic to the content review servers. 
```

Comment:
should this be shown in the default.sakai.properties file or can it be a hidden property?

Reply:
IMHO I think all properties should be documented. If there is a concern about bloating default.sakai.properties, I might suggest adding a comment in default.sakai.properties to direct readers to content-review/impl/turnitin-oc/README.md to find properties specific to turnitin-oc?